apps/client/components/form/SignInForm.tsx

```diff
@@ -45,16 +40,14 @@ const SignInForm = () => {
     },
   })
 
-
-
-
   async function onSubmit(values: z.infer<typeof FormSchema>) {
     const email = values.email
     const password = values.password
     const variables = { email, password }
 
-    const { data: { getUserByEmail } } = await getUser({ variables })
-
+    const {
+      data: { getUserByEmail },
+    } = await getUser({ variables })
```

Comment:
this looks wrong. I think you can just do `const fullName = await getUserByEmail(id)`

Reply:
unless there's a new way we're doing it?